input/fsh/research-consent-patient-profile.fsh

```diff
@@ -1,11 +1,21 @@
+Invariant: research-consent-patient-requirement
+Description: "A patient must have at least an email or phone number"
+Expression: "telecom.where(system='email').exists() or telecom.where(system='phone').exists()"
+Severity: #error
+
+Extension:      ResearchConsentPatientResearchSubject
```

Comment:
I am not following why we need this extension. Can we maybe leave a comment to provide more description? I think the subject of the consent should be captured in the `Consent.subject` and if the consent is a research consent it will be implied that the patient is subject of the research.